python/py-pew/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup			python 1.0
+PortGroup           github 1.0
+
+set package			pew
+set user			berdario
+version				1.1.0
+
+name				py-$package
+github.setup        $user $package $version
```

Comment:
Why do you need to do that?

Doesn't
```
github.user berdario pew 1.1.1
```
do exactly what you need? You can access those variables later if you need them, with:
* `${github.author}`
* `${github.project}`
* `${github.version}`
* `${github.tag_prefix}`

Reply:
I removed the cruft but kept the `name` line because I get this otherwise:

> Error: Portfile directory py-pew does not match port name pew